File: 2st/Riyeon/무작위로 K개의 수 뽑기.java
import java.util.*;
class Solution {
    public int[] solution(int[] arr, int k) {
        List<Integer> li = new ArrayList<>();
        
        for(int i=0; i<arr.length; i++){
            int cnt=0;
            for(int j=0; j<li.size(); j++)
                if(arr[i] == li.get(j))
                    cnt++;
            if(cnt==0 && li.size()<k)
                li.add(arr[i]);
        }
        
        while(li.size() < k){
            li.add(-1);
        }
            
        return li.stream().mapToInt(i->i).toArray();
    }
}

File: 2st/Riyeon/무작위로 K개의 수 뽑기.py
def solution(arr, k):
    answer = []
    i = 0
    while len(answer) < k:
        if len(arr) > i and arr[i] not in answer:
            answer.append(arr[i])
        elif len(arr) <= i:
            answer.append(-1)
        i += 1
    return answer